api/serverless-config-default.yml

```diff
@@ -493,14 +493,19 @@ functions:
                 method: GET
                 cors: true
     # Integrations
-    incrementalAriIngest:
+    incrementalAriIngestHttp:
         handler: src/components/integration/routes.incrementalAriIngest
         timeout: 900
         events:
-            - schedule:
-                rate: cron(0 5 ? * TUE *) # Every Tuesday at 5 a.m.
-                enabled: ${self:custom.scheduledAriIngestEnabled.${opt:stage}, false}
             - http:
```

Comment:
Does this need a http endpoint? Can this just be invoked via AWS CLI?

Reply:
The idea was to add an endpoint so that non-devs could trigger it without needing a dev to be involved.

Reply:
Ah, didn't realise non-devs needed this